api4/user_test.go

```diff
@@ -2068,8 +2069,19 @@ func TestUserLoginMFAFlow(t *testing.T) {
 	})
 
 	t.Run("WithCorrectMFA", func(t *testing.T) {
-		t.Skip("Skipping test that fails randomly.")
-		code := dgoogauth.ComputeCode(secret.Secret, time.Now().UTC().Unix()/30)
+		secret3, err := th.App.GenerateMfaSecret(th.BasicUser.Id)
```

Comment:
Nit, can we just call this `secret` since everything is scoped to this function?